ecosystem/node-checker/vfn-check-client/src/check.rs

```diff
@@ -97,6 +97,32 @@ pub enum SingleCheckResult {
     NoVfnRegistered(NoVfnRegistered),
 }
 
+#[derive(Debug, Serialize)]
+pub struct SingleCheckSuccess {
+    /// The evaluation summary returned by NHC. This doesn't necessarily imply
+    /// that the node passed the evaluation, just that an evaluation was returned
+    /// successfully and it passed the API available check.
+    pub evaluation_summary: EvaluationSummary,
+
+    /// This is the address that we used to get this successful evaluation.
+    /// This is presented in a normal URL format, not the NetworkAddress
+    /// representation. Example value for this field: http://vfn.site.com:8080.
+    /// Note, sometimes the address we started with was a DNS name, and we resolved
+    /// it to an IP address. As such, this IP address may become incorrect down
+    /// the line. In that case, refer to vfn_address in SingleCheck, or just
```

Comment:
Is SingleCheck in output too?

Reply:
Yeah. The output I pasted above is the entirety of the output you get (for each account address).